### ISSUE TITLE
[TW-1554] task < filter > start < mods > creates extra annotation

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1580]
_David Patrick on 2015-02-26T20:51:22Z says:_ 

 If I issue 
 
task 142 start notes on what I'm starting
 
task will create two annotations sumultaneously;
 
                02/26 15:34 notes on what I'm starting    
                02/26 15:34 start:
 
where I think the desirable outcome should be;
 
                02/26 15:34 start: notes on what I'm starting

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-02-26T20:51:22Z
Modified: 2017-01-16T17:21:27Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-01-16T17:21:16Z says:_ 

 Good suggestion.

**Comment by tbabej** [comment]
Fixed in 2.6.0.
